### DISCUSSION TITLE
How I keep running pocketbase like pm2 for nodejs in NginX/Ubuntu?

### DISCUSSION THREAD

**Discussion by aliahadmd** [pocketbase/pocketbase#2259] [Q&A] [answered]
Hi, I am trying to deploy Sveltekit and pocketbase. I am using NginX reverse proxy. I can keep running Sveltekit with "pm2". But How can I keep running Pocketbase! If i close terminal, Pocketbase server will stopping.
Thanks.

**Comment by ganigeorgiev** [comment, marked as answer]
You can run the PocketBase process in "background" (aka. something like `./pocketbase serve &`).

But a better approach would be to register a systemd service that will auto start/restart the PocketBase process (eg. in case of a server restart). For an example service file you can check "Step 4" of https://pocketbase.io/docs/going-to-production/#minimal-setup.

**Reply by aliahadmd** [reply]
Thank you so much. It works now! After many times I tried, I figured out it is` /etc/...` not `/lib/...`  to create pocketbase.service.

![image](https://user-images.githubusercontent.com/90032081/230947254-6059c17e-c647-4640-9a95-6a97a5ee061d.png)

**Comment by Karlstens** [comment]
I recently discovered both pm2 and pocketbase and started tinkering with both projects on my Windows desktop. 

Running pocketbase.exe with a config file works well whilst I learn the application - no doubt this method is loaded with red flags for anything beyond general learning. 😅 (keen for any tips with this)

pocketbase-pm2-config.json
```
{
	"apps": [
		{
			"name": "pocketbase",
			"script": "C:\\your_custom_path\\pocketBase\\pocketbase.exe",
			"watch": false,
			"autorestart": true,
			"max_restarts": 10,
			"exec_interpreter": "none",
			"exec_mode": "fork",
			"args": "serve"
		}
	]
}
```

Then in the bash terminal;

```
/c/your_custom_path/pocketbase
$ pm2 start pocketbase-pm2-config.json
```
